scripts/convert_access_to_config.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+import click
+import yaml
+
+
+@click.command(context_settings=dict(max_content_width=200))
+@click.argument("access_filename", default="access_log.yaml")
+@click.argument("config_filename", default="config.yaml")
+@click.option("--use-filenames", is_flag=True, help="Use filenames, not metadata.")
```

Comment:
this help doesn't really help me understand what --user-filenames does

Reply:
I will try and be more expansive. I was being extra terse because of the line wrapping (that is no longer an issue).